.konflux/dockerfiles/hub-ui.Dockerfile

```diff
@@ -1,6 +1,6 @@
 # --- builder image
-ARG NODEJS_BUILDER=registry.access.redhat.com/ubi8/nodejs-18:latest
-ARG RUNTIME=registry.access.redhat.com/ubi8/nginx-122:latest
+ARG NODEJS_BUILDER=registry.access.redhat.com/ubi9/nodejs-18:latest
+ARG RUNTIME=registry.access.redhat.com/ubi9/nginx-122:latest
```

Comment:
We should use digest for those (and let renovate update them).

Reply:
ya make sense
Updated with sha now